source/cloud-security/amazon/services/supported-services/custom-buckets.rst

```diff
@@ -128,8 +130,8 @@ These authentication methods require using the ``/root/.aws/credentials`` file t
 
 The available authentication configuration parameters are the following:
 
--  ``<aws_profile>``: A valid profile name from a Shared Credential File or AWS Config File with the permission to read logs from the bucket.
+-  ``<aws_profile>``: A valid profile name from a Shared Credential File or AWS Config File with permission to read logs from the bucket.
```

Comment:
Added links for `Shared Credential File` and `AWS Config File`. The `profile name` link would be the same used for `Shared Credential File` since the `.aws/credentials` file is mentioned inside the `Profiles` section. This way the user will read about both topics. 
In regards to the `"permissions to read logs from the bucket"`, these have been abstracted in the `/_templates/cloud/amazon/create_policy.rst`, `/_templates/cloud/amazon/bucket_policies.rst`, and `/_templates/cloud/amazon/attach_policy.rst` files present in many of the other services (`Config`, `Cloudtrail`, etc.). I'm adding a link that contains information about [Amazon S3 policy actions](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-with-s3-actions.html) to it.